crates/re_ui/src/list_item2/scope.rs

```diff
@@ -25,6 +59,7 @@ impl Default for LayoutStatistics {
         Self {
             max_desired_left_column_width: f32::NEG_INFINITY,
             is_action_button_used: false,
+            max_item_width: f32::NEG_INFINITY,
```

Comment:
Well spotted. I actually got bitten by a NEG_INF in `LayoutInfo` (which I fixed in #6211).

I'm going to replace those by `Option<f32>` to make the uninit state more explicit. (Will do that in #6211 because some stuff has already changed there.)